packages/contracts/deploy/new/00_managing-dao/00_managing-dao.ts

```diff
@@ -51,4 +51,4 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   });
 };
 export default func;
-func.tags = ['ManagingDao'];
+func.tags = ['ManagingDao'].concat('New');
```

Comment:
Why not simply a new array item?

```suggestion
func.tags = ['ManagingDao', 'New'];
```